contrib/firmware_packager/firmware_packager.py

```diff
@@ -38,23 +38,94 @@ def cd(path):
   <project_license>proprietary</project_license>
   <developer_name>{developer_name}</developer_name>
   <releases>
-    <release version="{release_version}" date="{date}">
+    <release urgency="{release_urgency}" version="{release_version}" date="{date}">
+      <checksum filename="firmware.bin" target="content"/>
       <description>
         <p>{release_description}</p>
+        {release_features_formatted}
       </description>
     </release>
   </releases>
   <custom>
     <value key="LVFS::VersionFormat">{version_format}</value>
     <value key="LVFS::UpdateProtocol">{update_protocol}</value>
   </custom>
+  <categories>
+{firmware_categories_formatted}
```

Comment:
Rather than formatting it in the code, can we do this in the template? e.g.

	{% for category in categories %}
	  <category>{{category}}</category>
	{% endfor %}